tests/functional/lib/mesh/climesh.go

```diff
@@ -374,8 +402,23 @@ func NewCLIMeshFromYaml(meshDefinition YamlData, dirSuffix string) (*CLIMesh, er
 		}
 	}
 
+	for k := range meshDefinition.Nodes {
```

Comment:
so here we fill in Nodedef, then copy it to nodedefbase, then do nodedef = nodedefbase + nodedefconnections

i think it would be more straightforward to change NewCLIMeshFromYaml to fill in NodedefBase to begin with, then do

nodedef = nodedefbase + nodedefconnections at the end

Reply:
I made the changes